terra_layer/permissions.py

```diff
@@ -3,4 +3,4 @@
 
 class LayerPermission(permissions.BasePermission):
     def has_permission(self, request, view):
-        return request.user.has_perm('django_geosource.can_manage_sources')
+        return request.user.has_perm('can_manage_layers')
```

Comment:
AFAIk, you need to prefix with the application name

Reply:
Done